src/components/Modal/Modal.tsx

```diff
@@ -70,32 +113,75 @@ export const BaseModal: ChakraComponent<
       headingSize: "heading4",
     });
 
-    return (
-      <ChakraModal
-        id={id}
-        isOpen={isOpen}
-        onClose={onClose}
-        scrollBehavior="inside"
-        size={size}
-        {...rest}
-      >
-        <ModalOverlay />
-        <ModalContent>
-          <ModalHeader>{finalTitle}</ModalHeader>
-          <ModalCloseButton />
-          <ModalBody>{bodyContent}</ModalBody>
+    if (
```

Comment:
@EdwinGuzman i'm not really describing it well but I mean you'll never hit this warning, you'll just have a type error and it'll display as a "default" modal, which is not exactly how we said errors should be handled. So maybe i need to change how this part works

Reply:
I see, yes, typescript will catch the issue in your editor or when the app is built. This is helpful, however, in applications that don't use typescript. One example is MLN (MyLibraryNyc) where these warnings do get logged.